app/init.php

```diff
@@ -1122,6 +1123,26 @@ function getDevice($root): Device
     }
 }
 
+/**
+ * Get database connection pool size for worker processes.
+ *
+ * @return int
+ * @throws \Exception
+ */
+function getWorkerPoolSize(): int
```

Comment:
Can't we just have the same logic for both and avoid the extra complexity at least for now? Also, we should try our best to avoid adding functions like this to the global scope.